css/index.css

```diff
@@ -1 +1,687 @@
+  border-bottom: 1px solid black;
+}
+
+.services .row {
+  justify-content: space-between;
+}
+
+.services .row .col-50 {
+  width: 48%;
+}
+
+.services-bottom-img {
+  width: 100%;
+}
+
+.services .list {
+  margin-top: 2rem;
+  padding-left: 2.27%;
+}
+
+.services .list .list-item {
+  margin-bottom: 1rem;
+}
+
+/* CARD */
+
+.card {
+  background-color: #f4f3f4;
+  margin-bottom: 3rem;
+  padding: 2rem;
+  font-size: 1.8rem;
+  border: 1px solid black;
+}
+
+.card__p {
+  margin-bottom: 2rem;
+}
+
+.btn--card,
+.button--product {
+  border: 1px solid black;
+  border-radius: 5px;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.6rem;
+  padding: 1rem 3rem;
+  cursor: pointer;
+  transition: all 200ms;
+}
+
+.btn--card:hover,
+.btn--product:hover {
+  background-color: #cbecff;
+}
+
+/* LINKS SECTION */
+.links {
+  text-align: center;
+}
+
+.links__list {
+  display: flex;
+  flex-wrap: wrap;
+  line-height: 1.7;
+}
+
+.links__list a,
+.links__list a:visited,
+.links__social-list a,
+.links__social-list a:visited {
+  color: #999;
+  font-family: 'Montserrat', sans-serif;
+  text-decoration: none;
+  margin-right: 1.14%;
+  transition: all 0.2s;
+}
+
+.links__list a:hover,
+.social-list a:hover {
+  color: #cbecff;
+}
+
+@media (max-width: 800px) {
+  .row:first-of-type {
+    margin-bottom: 0.8rem;
+  }
+
+  .services .row:first-of-type {
+    margin-top: 5rem;
+  }
+
+  /* NAVIGATION */
+  .nav {
+    height: 18rem;
+  }
+  .nav__container {
+    flex-direction: column;
+    justify-content: space-between;
+    padding: 3rem 0;
+  }
+  /* .nav-container,
+  .nav-links {
+    flex-direction: column;
+  } */
+  .nav__links {
+    width: 100%;
+    align-items: center;
+    justify-content: space-around;
+  }
+  .nav__link {
+    font-size: 1.4rem;
+  }
+  .nav__logo {
+    margin-left: 0;
+    order: -1;
+  }
+  /* HEADER */
+  .row--header {
+    padding: 18.8rem 0 6.7rem;
+  }
+
+  .page-container .header {
+    padding-bottom: 30px;
+  }
+
+  .heading-primary {
+    font-size: 7rem;
+  }
+
+  .cta {
+    width: 35%;
+    animation: none;
+    align-items: center;
+  }
+
+  .btn--cta {
+    font-size: 1.8rem;
+    padding: 0.5rem 1.5rem;
+    width: 60%;
+  }
+
+  .col-50--header {
+    width: 100%;
+  }
+
+  .header .col-50:nth-child(2) {
+    display: none;
+  }
+
+  .main--index,
+  .main-page {
+    width: 95%;
+    margin: 0 auto;
+  }
+
+  /* PAGES */
+  .page-container .header {
+    padding-top: 18rem;
+  }
+  .page-container .header h1,
+  .page-container .header p {
+    padding: 1rem;
+  }
+
+  /* SERVICES PAGE */
+  .services .row .col-50 {
+    width: 100%;
+  }
+
+  .card {
+    margin: 0 1rem 3rem;
+  }
+
+  .features,
+  .product {
+    width: 80%;
+    margin: 0 auto;
+  }
+
+  .features .col-33,
+  .product .col-33 {
+    margin-bottom: 2rem;
+  }
+}
+
+@media (max-width: 500px) {
+  /* GRID */
+  .col-50,
+  .col-33 {
+    width: 100%;
+  }
+  /* NAVIGATION */
+  .navigation {
+    position: relative;
```

Comment:
I'm not sure if you need to set position: relative here unless I'm missing something